src/Microsoft.ComponentDetection.Detectors/poetry/PoetryComponentDetector.cs

```diff
@@ -54,17 +54,15 @@ protected override async Task OnFileFoundAsync(ProcessRequest processRequest, ID
 
```

Comment:
Also updates to detectors that impact existing graphs should have a bump of the version, see `Version` on line 33.